lib/deck.rb

```diff
@@ -0,0 +1,25 @@
+require 'pry'
+class Deck
+
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.count
```

Comment:
Be careful when accessing instance variables, you want to always reference instance variables by `@`, even though Ruby lets us do it without it. This consistency will make more complex code easier to read later on!